pkg/maps/neighborsmap/neighborsmap.go

```diff
@@ -49,6 +49,7 @@ type Key6 struct {
 // +k8s:deepcopy-gen:interfaces=github.com/cilium/cilium/pkg/bpf.MapValue
 type Value struct {
```

Comment:
To avoid the {key,value}-size mismatch problems in the future, I think it makes sense to pass this and others recently introduced Go structs to the alignchecker.

Reply:
Uh, I'm pretty sure I had done that at some point in #10626. I must have removed those changes by mistake when I reorganized the commit history to bisect :-/

Anyway, I have added checks for the three new maps (neighborsmap, eventsmap, and signalmap) and double checked that other maps are already tested. I also reorganized the commits of the present pull request locally to make sure that the alignchecker detects the missing padding (by moving the fix before the alignchecker update).